integration_tests/test_borrow.py

```diff
@@ -1,7 +1,4 @@
-from nose.tools import (
```

Comment:
The tests in the integration_tests probably warrant some discussion. I haven't been able to run them for testing. There are a few changes from the automated runs, and find replace, but nothing in the `integration_tests` folder has been tested.

Reply:
Most of the files in integration_tests could be removed. I think the combination of self-test-client (for diagnosing problems) and our mobile-based integration test suites (for everyday testing) negate the need for most of these scripts.

The one exception is test_search.py, which has a lot of knowledge that isn't contained anywhere else -- knowledge about what search requests ought to return what sorts of books.

I'm OK with keeping test_search.py without modifying it; we can fix it up later the next time we need to run search benchmarks. And I'm OK with removing the rest of the integration tests since we have other means of testing that stuff.

Some automatically-run integration tests that verify behavior on various CMs without relying on the mobile app would be useful, but I don't think the existing code (which is quite old) is a good starting point for that project.

Reply:
For now, leave integration_tests alone, and we'll decide to either fix or remove later.